PathPlanning/BreadthFirstSearch/bfs.py

```diff
@@ -0,0 +1,270 @@
+"""
+
+Breadth-First grid planning
+
+author: Erwin Lejeune (@spida_rwin)
+
+See Wikipedia article (https://en.wikipedia.org/wiki/Breadth-First_search)
+
+"""
+
+import math
+
+import matplotlib.pyplot as plt
+
+import random
+
+show_animation = True
+
+
+class BFSPlanner:
```

Comment:
BFSPlanner -> BreadthFirstSearchPlanner is better

Reply:
Done.